pkg/controller/manager/manager_controller.go

```diff
@@ -264,183 +289,208 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
-			r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Waiting for secret '%s' to become available", secretName), nil, reqLogger)
+			logc.Info(fmt.Sprintf("Waiting for secret '%s' to become available", secret))
+			r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Waiting for secret '%s' to become available", secret), nil, logc)
 			return reconcile.Result{}, nil
 		}
 		trustedBundle.AddCertificates(certificate)
 	}
-	certificateManager.AddToStatusManager(r.status, render.ManagerNamespace)
+	certificateManager.AddToStatusManager(r.status, request.InstallNamespace())
 
 	// Check that Prometheus is running
+	// TODO: We'll need to run an instance of Prometheus per-tenant? Or do we use labels to delimit metrics?
+	//       Probably the former.
 	ns := &corev1.Namespace{}
 	if err = r.client.Get(ctx, client.ObjectKey{Name: common.TigeraPrometheusNamespace}, ns); err != nil {
 		if errors.IsNotFound(err) {
-			r.status.SetDegraded(operatorv1.ResourceNotFound, "tigera-prometheus namespace does not exist Dependency on tigera-prometheus not satisfied", nil, reqLogger)
+			r.status.SetDegraded(operatorv1.ResourceNotFound, "tigera-prometheus namespace does not exist Dependency on tigera-prometheus not satisfied", nil, logc)
 		} else {
-			r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying prometheus", err, reqLogger)
+			r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying prometheus", err, logc)
 		}
 		return reconcile.Result{}, err
 	}
 
-	pullSecrets, err := utils.GetNetworkingPullSecrets(installation, r.client)
+	pullSecrets, err := utils.GetNetworkingPullSecrets(request.installation, r.client)
 	if err != nil {
 		log.Error(err, "Error with Pull secrets")
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Error retrieving pull secrets", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Error retrieving pull secrets", err, logc)
 		return reconcile.Result{}, err
 	}
 
 	esClusterConfig, err := utils.GetElasticsearchClusterConfig(context.Background(), r.client)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			r.status.SetDegraded(operatorv1.ResourceNotFound, "Elasticsearch cluster configuration is not available, waiting for it to become available", err, reqLogger)
+			r.status.SetDegraded(operatorv1.ResourceNotFound, "Elasticsearch cluster configuration is not available, waiting for it to become available", err, logc)
 			return reconcile.Result{}, nil
 		}
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get the elasticsearch cluster configuration", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get the elasticsearch cluster configuration", err, logc)
 		return reconcile.Result{}, err
 	}
 
 	// Get secrets used by the manager to authenticate with Elasticsearch.
+	// TODO: esSecrets should be namespaced in the tenant's namespace. We can't get rid of this 100%
+	// since some secrets are needed for Kibana auth flow.
```

Comment:
For management clusters that don't have Kibana enabled, we can skip over this entirely.